src/modality-specific-files/magnetic-resonance-spectroscopy.md

```diff
@@ -179,19 +179,7 @@ A guide for using macros can be found at
 
 ### MRS-relevant fields
 
-Metadata fields that MUST be present:
-
-<!-- This block generates a metadata table.
-These tables are defined in
-  src/schema/rules/sidecars
-The definitions of the fields specified in these tables may be found in
-  src/schema/objects/metadata.yaml
-A guide for using macros can be found at
- https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
--->
-{{ MACROS___make_sidecar_table("mrs.MRSRequiredFields") }}
-
-SHOULD be present:
+Metadata fields that MUST, SHOULD or MAY be present:
```

Comment:
Combined this into one table. Let me know if you think this is harder to read and we should split it back out into three.

Reply:
I think having them in three makes it easier to parse/read.

Reply:
Will revert.

Reply:
Done.